source/functions/BEMIO/readCAPYTAINE.m

```diff
@@ -27,7 +30,15 @@
 p = waitbar(0,'Reading Capytaine netcdf output file...'); %Progress bar
 
 hydro(F).code = 'CAPYTAINE';
-[meshdir,name,~] = fileparts(filename);
+
+[base_dir, name, ~] = fileparts(filename);
+
+if(~isempty(hydrostatics_sub_dir))
+    hydrostatics_dir = append(base_dir, hydrostatics_sub_dir);
```

Comment:
Last thought, can you add to the logic here so that user's old BEMIO scripts can function without the ``hydrostatics_sub_dir`` argument? Maybe something like ``if exist('hydrostatics_sub_dir','var') & ~isempty(hydrostatics_sub_dir)``. I believe that should allow users to not input the directory, but the single '&' should not break the second condition if the variable doesn't exist.

Reply:
sure 👍